controllers/teams.js

```diff
@@ -17,6 +18,20 @@ moment.locale('de');
 
 const OPTIONAL_TEAM_FEATURES = ['rocketChat', 'videoconference', 'messenger'];
 
+handlebars.registerHelper('stringify', (jsonValue) => {
+	const attributes = [];
+	const escapedKeyList = ['style', 'data-payload'];
+	const stringifyValue = JSON.stringify(jsonValue);
+	const escapedValueList = ['display: none', stringifyValue];
+	escapedKeyList.forEach((entity, i) => {
+		const escapedKey = handlebars.escapeExpression(entity);
+		const escapedValue = handlebars.escapeExpression(escapedValueList[i]);
+		attributes.push(`${escapedKey}="${escapedValue}"`);
+	});
+	const escapedOutput = `<td ${attributes.join(' ')} />`;
+	return new handlebars.SafeString(escapedOutput);
+});
```

Comment:
shouldn't be done as an helper. Please do this directly in handlebars. You can write a handlebar component if noone exists. Also Stringify is a to regular name to do something with tables